man/systemd-sleep.conf.xml

```diff
@@ -78,9 +78,20 @@
         <term>suspend-then-hibernate</term>
 
         <listitem><para>A low power state where the system is initially suspended
-        (the state is stored in RAM). If not interrupted within the delay specified by
-        <command>HibernateDelaySec=</command>, the system will be woken using an RTC
-        alarm and hibernated (the state is then stored on disk).
+	(the state is stored in RAM) and then hibernated based on battery percentage
```

Comment:
indentation off?

Reply:
fixed

Reply:
Not fixed...

Reply:
i am still working on some changes. for the same reason i hv marked PR as draft and not pushed since LP reviewed. please wait for me to remove it from draft :)
i comment for keeping track for myself what is fixed n what is pending :)

Reply:
Sorry, I thought you made a change.

Reply:
No. It's fine. :)